src/libs/notifications.js

```diff
@@ -58,13 +50,14 @@ const NotificationDisplay = Utils.connectStore(notificationStore, 'notificationS
     const onLast = notificationNumber + 1 === notifications.length
 
     const { title, message, detail, type } = notifications[notificationNumber]
-    const color = Utils.cond(
-      [type === 'success', colors.success(1.5)],
-      [type === 'info', colors.dark()],
-      [type === 'welcome', colors.accent()],
-      [type === 'warn', colors.warning(1.2)],
-      [type === 'error', colors.danger()]
-    ) || colors.dark()
+    const baseColor = Utils.cond(
+      [type === 'success', () => colors.success],
+      [type === 'info', () => colors.accent],
+      [type === 'welcome', () => colors.accent],
+      [type === 'warn', () => colors.warning],
+      [type === 'error', () => colors.danger],
+      () => colors.accent
+    )
     const iconType = Utils.cond(
       [type === 'success', 'success-standard'],
       [type === 'warn' || type === 'error', 'warning-standard'],
```

Comment:
Mocks show error as an exclamation point in a circle while warning is an exclamation point in a triangle.